packages/core/core-flows/src/promotion/steps/add-or-remove-campaign-promotions.ts

```diff
@@ -0,0 +1,64 @@
+import { ModuleRegistrationName } from "@medusajs/modules-sdk"
+import { IPromotionModuleService, LinkWorkflowInput } from "@medusajs/types"
+import { StepResponse, WorkflowData, createStep } from "@medusajs/workflows-sdk"
+
+export const addOrRemoveCampaignPromotionsStepId =
+  "add-or-remove-campaign-promotions"
+export const addOrRemoveCampaignPromotionsStep = createStep(
```

Comment:
Maybe a silly oversight from my side, but is this step used anywhere after we made the split?

Reply:
no, because if one of them fails, we need to revert the other. Its easier to handle revert scenario when its in separate steps.

Reply:
Yeah, that's what I meant 😄 

This file is no longer used, so we can delete it right?